### ISSUE TITLE
LDAP User Login Error and Information Details

### ISSUE THREAD

**Issue by ushioTora** [RocketChat/Rocket.Chat#18561]
### Description:

Cant Login LDAP User and Unable to manage ldap user. When you click the user info only white blank page appear

### Steps to reproduce:

1. Go to administration >
2. Click Users
3. Click LDAP Users

### Expected behavior:

I was hoping to see details of information such as name, password, edit button, etc.

### Actual behavior:
![image](https://user-images.githubusercontent.com/13395611/90263820-c4f92b00-de82-11ea-9e90-dc57b757aac6.png)

### Server Setup Information:

- Version of Rocket.Chat Server: 3.5.2
- Operating System: Censto 7
- Deployment Method: tar
- Number of Running Instances: 1
- DB Replicaset Oplog: Enabled
- NodeJS Version:  v12.18.3
- MongoDB Version: 4.0.19

### Client Setup Information

- Desktop App or Browser Version:  Rocket Chat App v2.17.11 | Brave v1.11 , Firefox v79.0
- Operating System: Windows 10

### Relevant logs:

![rcerror2](https://user-images.githubusercontent.com/13395611/90265197-ccb9cf00-de84-11ea-9309-986f9f9b0462.png)
![rcerror1](https://user-images.githubusercontent.com/13395611/90265208-cfb4bf80-de84-11ea-92a5-383269af1115.png)

**Comment by ZeroPoke** [comment]
I noticed on my install I did today.  That users from LDAP didnt have roles, and ones without roles I couldnt access the user info.

**Comment by ushioTora** [comment]
Yes , I noticed that one too. Right now I'm trying to figure out how to set roles on LDAP users upon authentication. Please let me know if you've already fixed this issue. Thanks

**Comment by jeffrey-butler** [comment]
Is there a way to automatically assign roles? When running the sync it looks to have assigned a role to one user, while the reset remained blank

**Comment by dieug** [comment]
yes, we have it effect too.
in rocketchat 3.5.1 3.5.2 3.5.3 3.5.4 it exists.

**Comment by ushioTora** [comment]
Still the same after upgrading to 3.6.0 Can't edit the user. Unless you do it manually per user by going to http://domainorserver:3000/administration/users/userid  :( 
Tested on my 2 servers. I thought i was having a problem on my Active Directory but it seems ok

**Comment by dieug** [comment]
i can add, when i had change a "ldap group dn" to another level of structure (as i remember, i did point it to root dn), all users became admins (had role "admin" in profiles) after import.
